django/db/backends/mysql/operations.py

```diff
@@ -30,6 +32,17 @@ class DatabaseOperations(BaseDatabaseOperations):
     cast_char_field_without_max_length = 'char'
     explain_prefix = 'EXPLAIN'
 
+    not_supported_distinct_aggregates = (
+        aggregates.Variance,
+        aggregates.StdDev,
+    )
+
+    def check_expression_support(self, expression):
```

Comment:
Did you consider putting this in the base class? You can use `connection.display_name` for MySQL/Oracle.

Reply:
+1 as seems more DRY. 3rd party backends would also only need to add `unsupported_distinct_aggregates` and not implement `check_expression_support()`.